hardhat.config.ts

```diff
@@ -28,17 +28,22 @@ const config: HardhatUserConfig = {
     deployer: {
       default: 0
     },
+    defenderRelayer: {
+      default: 0
+    },
+    executiveTeam: {
+      default: 0,
+    },
     owner: {
       default: 0,
       rinkeby: "0x72c9aA4c753fc36cbF3d1fF6fEc0bC44ad41D7f2",
       fuji: "0x0fB374787B0bB7e62bD82A1B020b12fa239aB788"
     },
     manager: {
```

Comment:
let's be explicit- these are defender relayers so let's put them under the 'defenderRelayer' named accounts